.brv/context-tree/development/ito_workflow/_index.md

```
@@ -1,88 +1,73 @@
+## Drill-down guide
+Use the child entries for detail on:
+- mirror safety and publishing: **published_ito_mirror.md**
+- audit branch concurrency and retention: **audit_mirror_concurrency_and_temp_naming.md**
+- coordination bootstrap and ref behavior: **coordination_branch_git_behavior.md**
+- `.ito/` symlink repair and sync: **coordination_symlink_repair_and_sync.md**
+- config scope and storage mode: **ito_config_gotcha.md**
+- read-only worktree validation: **worktree_validation_flow.md**
+- installer cleanup of renamed assets: **obsolete_specialist_cleanup.md**
+- domain discovery gates: **ddd_discovery_workflow.md**
+- orchestration source-of-truth: **ito_orchestration_consolidation.md**
+- pre-push adversarial review: **pre_push_adversarial_code_review.md**
```

markdownlint-cli2 (0.22.1)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)